Misc/NEWS.d/next/Core and Builtins/2020-02-13-01-30-22.bpo-39573.uTFj1m.rst

```diff
@@ -0,0 +1,2 @@
+Add :c:func:`Py_IS_TYPE` static inline function to check whether
```

Comment:
Can you please check if the following formatting is rendered properly in HTML? (``make -C Doc venv html``).

```suggestion
Add :c:func:`Py_IS_TYPE(o, type)` static inline function to check whether
```